backend/src/gpml/handler/review.clj

```diff
@@ -37,29 +37,36 @@
                 {:topic-type topic-type :topic-id topic-id})]
     (resp/response review)))
 
-(defn new-review [db mailjet-config topic-type topic-id reviewer-id assigned-by]
+(defn create-review-with-multiple-reviewers [topic-type topic-id assigned-by conn reviewer-id]
```

Comment:
the function name doesn't relate with the logic inside nor the args received ... the logic inside only create a review with a reviewer